<launch>

  <rosparam file="$(find robot_control)/config/cartesian_impedance_controller.yaml" command="load" ns="dsr01"/>

  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
    output="screen" ns="dsr01" args="CartesianImpedance_trajectory_controller joint_state_controller"/>
  
  <node name="controller_manager" pkg="controller_manager" type="controller_manager" respawn="false" output="screen" />
  
  <node name="config_impedance_controller" pkg="robot_control" type="config_impedance_controller.py" output="screen" />

</launch>